SampleApps/WebView2APISample/AppWindow.cpp

```diff
@@ -1804,10 +1804,10 @@ void AppWindow::RegisterEventHandlers()
                     [this]
                     {
                         std::wstring message =
-                            L"We detected there is a new version for the browser.";
+                            L"A new version of Microsoft Edge is available on this machine. ";
```

Comment:
```suggestion
                            L"A new version of Microsoft Edge WebView2 is available on this machine. ";
```
WebView2 or Edge browser?

Reply:
probably will go with:
A new version of the Microsoft Edge WebView2 Runtime is available on this machine.

Reply:
new Files Changed tab shows:
                            L"A new version of the Microsoft Edge WebView2 Runtime is available on this machine. ";